ports/nrf/hal/hal_twi.c

```diff
@@ -29,6 +29,8 @@
 
 #ifdef HAL_TWI_MODULE_ENABLED
 
+#define HAL_TWI_ERROR_CLR_MASK (7)
```

Comment:
You could replace this with `TWI_ERRORSRC_DNACK_Msk | TWI_ERRORSRC_ANACK_Msk | TWI_ERRORSRC_OVERRUN_Msk`

Reply:
great idea!